socorro/external/crashstorage_base.py

```diff
@@ -253,368 +242,65 @@ def remove(self, crash_id):
         raise NotImplementedError("remove is not implemented")
 
 
-class PolyStorageError(Exception, MutableSequence):
```

Comment:
I removed all the poly-storage stuff and adjusted the processor so it always supports multiple crash destinations. That's a lot easier than having this abstraction layer with the complex poly-storage-error handling.